src/licensedcode/data/licenses/metamail.LICENSE

```diff
@@ -0,0 +1,21 @@
+---
+key: metamail
+short_name: metamail License
+name: metamail License
+category: Permissive
+owner: Bellcore
+spdx_license_key: metamail
+other_urls:
+    - https://github.com/Dual-Life/mime-base64/blob/master/Base64.xs#L12
+---
+
```

Comment:
This replaces [historical_9.RULE](https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/historical_9.RULE)